docs/en/development/continuous-integration.md

```diff
@@ -1,215 +1,198 @@
 ---
 toc_priority: 62
-toc_title: Continuous Integration Checks
+toc_title: 持续集成检查
```

Comment:
You've accidentially replaced `en` file with `zh` translation.

Reply:
Fix it